src/docs/releases/2.0.0.md

```diff
@@ -314,10 +314,11 @@ and removed the following methods:
 
 A new `SiteDisplayDriver` base class was introduced to simplify the process of adding a UI settings.
  
-
  !!! note
     With the new `SiteDisplayDriver` base class for custom settings, you no longer need explicit checks like `context.GroupId.Equals(GroupId, StringComparison.OrdinalIgnoreCase)` as the base class handles this check for you.
 
+A new extension has been introduced to simplify the registration of site setting drivers. Instead of using `service.AddScoped<IDisplayDriver<ISite>, CustomSettingsDisplayDriver>()`, you can now register it with the more streamlined `service.AddSiteDisplayDriver<CustomSettingsDisplayDriver>()`.
```

Comment:
```suggestion
A new extension has been introduced to simplify the registration of site-setting drivers. Instead of using `service.AddScoped<IDisplayDriver<ISite>, CustomSettingsDisplayDriver>()`, you can now register it with the more streamlined `service.AddSiteDisplayDriver<CustomSettingsDisplayDriver>()`.
```

Reply:
no need to compound the two words here

Reply:
Ok